docs/resources/Audit_Log.md

```diff
@@ -0,0 +1,143 @@
+| permission_overwrites | [channel](#DOCS_CHANNEL/channel-object) | array of [channel overwrite](#DOCS_CHANNEL/overwrite-object) objects| permissions on a channel changed |
+| nsfw | [channel](#DOCS_CHANNEL/channel-object) | bool | channel nsfw restriction changed |
+| application_id | [channel](#DOCS_CHANNEL/channel-object)| snowflake | application id of the added or removed webhook or bot |
+| permissions | [role](#DOCS_GUILD/role-object)| integer | [permissions](#DOCS_PERMISSIONS/permissions-bitwise-permission-flags) for a role changed |
+| color | [role](#DOCS_PERMISSIONS/role-object) | integer | role color changed |
+| hoist | [role](#DOCS_PERMISSIONS/role-object) | bool | role is now displayed/no longer displayed separate from online users |
+| mentionable | [role](#DOCS_PERMISSIONS/role-object) | bool | role is now mentionable/unmentionable |
+| allow | [role](#DOCS_PERMISSIONS/role-object) | integer | a permission on a text or voice channel was allowed for a role |
+| deny | [role](#DOCS_PERMISSIONS/role-object) | integer | a permission on a text or voice channel was denied for a role |
+| code | [invite](#DOCS_INVITE/invite-object) | string | invite code changed |
+| channel_id | [invite](#DOCS_INVITE/invite-channel-object) | snowflake | channel for invite code changed |
+| inviter_id | [invite](#DOCS_INVITE/invite-metadata-object) | snowflake | person who created invite code changed |
+| max_uses | [invite](#DOCS_INVITE/invite-metadata-object) | integer | change to max number of times invite code can be used |
+| uses | [invite](#DOCS_INVITE/invite-metadata-object) | integer | number of times invite code used changed |
+| max_age | [invite](#DOCS_INVITE/invite-metadata-object) | integer | how long invite code lasts changed |
+| temporary | [invite](#DOCS_INVITE/invite-metadata-object) | bool | invite code is temporary/never expires |
+| deaf | [user](#DOCS_USER/user-object) | bool | user server deafened/undeafened |
+| mute | [user](#DOCS_USER/user-object) | bool | user server muted/unmuteds |
+| nick | [user](#DOCS_USER/user-object) | string | user nickname changed |
+| avatar_hash | [user](#DOCS_USER/user-object) | string | user avatar changed |
+| id | any | snowflake | the id of the changed entity - sometimes used in conjunction with other keys |
+| type | any | string/int | type of entity created - can be of [channel type](#DOCS_CHANNEL/channel-object-channel-types) or a string like "role" |
+
+
+## Get Guild Audit Log % GET /guilds/{guild.id#DOCS_GUILD/guild-object}/audit-log
+
+Returns an an [audit log object](#DOCS_AUDIT_LOG/audit-log-object) for the guild. Requires the 'VIEW_AUDIT_LOG' permission.
+
+###### Query String Parameters
+
+|Field | Type | Description |
+| ---- | ----- | ---------- |
+| user_id | snowflake | filter the log for a user id |
+| action_type | int  | the type of [audit log event](#DOCS_AUDIT_LOG/audit-log-entry-object-audit-log-events) |
+| before | snowflake | filter the log before a certain entry id |
```

Comment:
this reminds me. i should really merge that before, after & around PR to add support for those values.